### PR TITLE
Just cleaned up some spelling while reading through the docs

### PR DESCRIPTION
**Pull request by RobbyB97**
modified:   Goals.adoc
	modified:   Introduction.adoc
	modified:   Manual.adoc
	modified:   PartSettings.adoc
	modified:   PresetsAndBanks.adoc

### CODE REVIEW

File: PartSettings.adoc [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
-To access the part settings, click on the "Part Settings" button in the sidebar 
+To access the part settings, click on the "Part Settings" button in the sidebar
 
 image::imgs/Selection_440.png[]
 
 === What are parts?
 
-ZynFusion is a multitimbral synthesizer - that means it can perform with multiple patches (timbres) at the same time. All 16 parts function independently and by default respond to respective MIDI channels. Sound output from all part is mixed to a stereo sum.
+ZynFusion is a multi-timbre synthesizer - that means it can perform with multiple patches (timbres) at the same time. All 16 parts function independently and by default respond to respective MIDI channels. Sound output from all part is mixed to a stereo sum.
```
Comment (fundamental) [inline PartSettings.adoc]: no hyphen is required.

File: PresetsAndBanks.adoc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 === Presets
 
 * [x] What are presets?
-* [ ] What types of presets we have in Zyn? (XMZ, XIZ, microtonal, MIDI bindings, ...)
+* [ ] What types of presets we have in Zyn? (XMZ, XIZ, micro-tonal, MIDI bindings, ...)
```
Comment (fundamental) [inline PresetsAndBanks.adoc]: No hyphen is required.

File: PartSettings.adoc (hunk 1 above)
Comment (RobbyB97) [reply PartSettings.adoc]: Fixed it :)

**Comment by fundamental** [comment]
Just as a followup, it looks like one of the two review comments were addressed. As soon as the other change request is addressed, this should be ready to merge. If you're unsure what I'm referring to, let me know and I should be able to merge and add the small change needed myself.

Thanks for your time.

**Comment by RobbyB97** [comment]
I thought you meant the multi-timbre and micro-tonal, so I changed those.

**Comment by fundamental** [comment]
Something must have gotten mixed up in one of the merge commits. I've merged and fixed the remaining issue.

**Pull request merged**